stubs/tree-sitter-languages/METADATA.toml

```diff
@@ -1,3 +1,9 @@
 version = "1.10.*"
 upstream_repository = "https://github.com/grantjenks/py-tree-sitter-languages"
 requires = ["tree-sitter>=0.20.3"]
+
+[tool.stubtest]
+# This package is unmaintained and doesn't support Python 3.13.
+# See https://github.com/grantjenks/py-tree-sitter-languages/issues/75 and
+# https://github.com/grantjenks/py-tree-sitter-languages/blob/main/README.rst#status
+skip = true
```

Comment:
It's a _bit_ weird that the "this is unmaintained" note was PR'd by the author of the competing package and merged without comment: https://github.com/grantjenks/py-tree-sitter-languages/pull/84. But it's true that it can't be installed on Python 3.13 currently and hasn't had any significant updates since February 2024.

The stubs were added by @Akuli in https://github.com/python/typeshed/pull/8548 and IIRC he's contributed to the upstream package as well